templates/zerver/find_account.html

```diff
@@ -28,7 +28,8 @@ <h1 class="get-started">{{ _("Find your Zulip accounts") }}</h1>
                     <li>{{ email }}</li>
                     {% endfor %}
                 </ul>
-
+                If you don't receive an email, you can
+                <a href = "{{current_url()}}" >find accounts for another email address</a>.
```

Comment:
This should be tagged for translation.

See https://zulip.readthedocs.io/en/latest/translating/internationalization.html for documentation on how to do that properly.

Reply:
@timabbott Done with the changes 
Also could you please check out #28598